app/tests/spec/views/base.js

```diff
@@ -40,7 +38,7 @@ function (chai, $, sinon, BaseView, p, Translator, EphemeralMessages, Metrics,
     var translator;
     var metrics;
     var fxaClient;
-    var relier;
+    var relier; //eslint-disable-line no-unused-vars
```

Comment:
`relier` is not used in this file, so we can remove it and remove `Relier` from the module list so that amdcheck passes.